lib/alamo.js

```diff
@@ -729,36 +841,53 @@ function parse_command_args(arg) {
 }
 
 let deploying = false;
-async function deploy(pg_pool, space_name, app_name, formation_type, image, command, port, healthcheck, features, labels, filters) {
+async function deploy(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Async function 'deploy' has a complexity of 9. Maximum allowed is 8.](https://app.codacy.com/manual/Akkeris/controller-api/pullRequest?prid=5244943)